encryption.rst

```diff
@@ -124,11 +124,9 @@ specific flags like so:
 .. code::
 
    # Generate a key pair
-   $ ssh-keygen -t rsa -b 2048
+   $ ssh-keygen -t rsa -b 4096 -m pem -N ''
```

Comment:
According to the man page and my testing, the `-m pem` option has no effect with that set of options, so please remove it.

Reply:
I changed my mind, it does make a difference on ubuntu